### PR TITLE
Conference website

### PR DESCRIPTION
**Pull request by ginabeki**
### Conference Website for Art Week Event
I performed the following things in this task;

- Built the home page.
- Built the about page and linked it to the home page.
- Created the two pages in two different versions screen sizes:
  - Mobile: up to 768px wide.
   - Desktop: 768px or wider.
- I also added Interactions on the page by linking the menu to the about page and header to the home page
- On mobile, I ensured that when the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page.
- When a user clicks the (X) button, the menu closes.
- Created featured speakers dynamically using JS.
Here's a [link](https://www.loom.com/share/1186e713f5884bf2bcc7b744adeba7f7) to my presentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,85 @@
...
+- Have a local version control like Git. Which is a open source distributed version control system designed for source code management.
+- A text editor (e.g Visual Studio Code, Vim, Atom & Sublime.)
+- A web browser (e.g Chrome, Safari, Mozilla Firefox.)
+- Live Server which is a web extension that helps you to live reload feature for dynamic content.
+- Install wave extension to check project accessibilty.
+
+### Live Demo
+
+- To view a live demo of the project you can click on this [link](https://ginabeki.github.io/Conference-Website/).
+
```
Comment (zieeco) [inline README.md]: - Hey, please let's add the link to the video presentation here in the readme. Also, your video is longer than 5 min in length as this is meant to not exceed 5 min long according to the project requirements. Please, kindly record another one with a maximum length of not more than 5 mins long to meet the project requirement.

File: about.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+          <li class="nav-link"><a href="">Join</a></li>
+          <li class="nav-link"><a href="">Sponsor</a></li>
+          <li class="nav-link"><a href="">News</a></li>
+          <li class="nav-link"><a href="">Campaign</a></li>
+        </ul>
+      </nav>
+    </header>
+    <section class="hero-about">
+      <h3 class="about-text">"Hello, Creative World!"</h3>
+      <h1 class="about-detail">Explore Kenyan Art</h1>
```
Comment (zieeco) [inline about.html:48]: - Hey, To enforce the best practices in using the html5 semantic tag, a webpage is only meant to have one h1 which denotes the main heading in the webpage. You can use another header tag here as your main heading is in the index.html file. Please update this to improve the web accessibility of your page.

File: css/styles.css [hunk 3]
```diff
@@ -0,0 +1,602 @@
...
+.individual {
+  display: flex;
+  gap: 15px;
+}
+
+.team-details {
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
```
Comment (zieeco) [inline css/styles.css]: - [ ] Hey, please let's improve our design in the `mobile version` a little to be close to the design template as the aim of this project is to help make a total clone of an existing design, thereby paying attention to details is a skill that will make you stand out in the job market. see the suggestion below:

*original design*
Screenshot from 2022-10-19 08-43-13

*your design*
Screenshot from 2022-10-19 08-45-03

*your design after I edited*
Screenshot from 2022-10-19 08-52-53

- As you can from those designs, the content on your design is not that close compared to the original design
   - You can reduce the font size,
   - Remove the gap in `.team-details`,
   - You can make the height/width of the image `8rem`,
   - You can also reduce the description of the team members,*[If you do this, your design will be very close :100:`]*
   - All these are just little tweaks around to improve the design.
- [x] Please, make a few adjustments to improve your design and also, to improve the user experience of your website.

File: css/styles.css [hunk 4]
```diff
@@ -0,0 +1,602 @@
...
+
+.card {
+  background-color: #b3b3b351;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  gap: 0.5rem;
+  width: 100%;
+  padding: 0.5rem;
+}
```
Comment (zieeco) [inline css/styles.css:211]: - Hey, I really like your design. well done! I just want to point out something. Please, let's try to be attentive to details. This skill is very important in the dev world. Your card is 3, while the design is 5, can we add more cards to match the number? Please, I encourage you to do this.

Screenshot from 2022-10-19 09-16-00

File: index.html [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+        <ul class="menu">
+          <li class="nav-link"><a href="./about.html">About</a></li>
+          <li class="nav-link"><a href="">Program</a></li>
+          <li class="nav-link"><a href="">Join</a></li>
+          <li class="nav-link"><a href="">Sponsor</a></li>
+          <li class="nav-link"><a href="">News</a></li>
+          <li class="nav-link"><a href="">Campaign</a></li>
+        </ul>
+      </nav>
+    </header>
```
Comment (zieeco) [inline index.html]: - Hey, you're doing great! Please, your header is missing the first top nav. Please, let's add this match the design as much as we can below:

*original design*
Screenshot from 2022-10-19 09-22-21

*your design*
Screenshot from 2022-10-19 09-21-36

File: about.html [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+        <ul class="menu">
+          <li class="nav-link"><a href="/index.html">Home</a></li>
+          <li class="nav-link"><a href="">Program</a></li>
+          <li class="nav-link"><a href="">Join</a></li>
+          <li class="nav-link"><a href="">Sponsor</a></li>
+          <li class="nav-link"><a href="">News</a></li>
+          <li class="nav-link"><a href="">Campaign</a></li>
+        </ul>
+      </nav>
+    </header>
```
Comment (zieeco) [inline about.html]: - Hey, you're doing great! Please, your header is missing the first top nav. Please, let's add this match the design as much as we can below:

*original design*
Screenshot from 2022-10-19 09-22-21

*your design*
Screenshot from 2022-10-19 09-21-36

File: css/styles.css (hunk 4 above)
Comment (zieeco) [reply css/styles.css:211]: - Also, the hover color of the card is `white`, let's update this also.

File: css/styles.css [hunk 7]
```diff
@@ -0,0 +1,602 @@
...
+  }
+
+  .detail {
+    width: 60%;
+    text-align: left;
+  }
+
+  .workshop {
+    flex-direction: row;
+  }
```
Comment (zieeco) [inline css/styles.css]: - Hey, to control the width of each workshop, let's add a widht of about 65% to almost be close to the design template, thereby aligning the card in the center below:

Screenshot from 2022-10-19 09-32-31

File: css/styles.css [hunk 8]
```diff
@@ -0,0 +1,602 @@
...
+}
+
+.workshop-1:hover {
+  background: linear-gradient(rgba(255, 255, 255, 0.1), rgba(255, 255, 255, 0.1)), url(../images/workshop-1.png) no-repeat center;
+}
+
+.workshop-2 {
+  background: linear-gradient(rgba(236, 82, 66, 0.7), rgba(39, 42, 49, 0.6)), url(../images/workshop-2.png) no-repeat center;
+  background-size: cover;
+  height: 10rem;
```
Comment (zieeco) [inline css/styles.css]: - Hey, please, let's increase the height to 12rem so our design is close to match the design template.

File: css/styles.css [hunk 9]
```diff
@@ -0,0 +1,602 @@
...
+  align-items: center;
+  width: 100%;
+  gap: 20px;
+  margin-top: 2rem;
+}
+
+.workshop-1 {
+  background: linear-gradient(rgba(236, 82, 66, 0.7), rgba(39, 42, 49, 0.6)), url(../images/workshop-1.png) no-repeat center;
+  background-size: cover;
+  height: 10rem;
```
Comment (zieeco) [inline css/styles.css]: - Hey, please, let's increase the height to 12rem so our design is close to matching the design template.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @ginabeki,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Good code organisation :heavy_check_mark: 


## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: css/styles.css (hunk 4 above)
Comment (ginabeki) [reply css/styles.css:211]: Done

File: index.html (hunk 5 above)
Comment (ginabeki) [reply index.html]: Done

**Comment by ginabeki** [comment]
The following changes were made;
- Added a new loom [live link](https://www.loom.com/share/87858677e9c740abbdf4e7646deae94a) that's within the required length.
- Removed extra h1 tag on the page to improve SEO
- Fixed the speaker's section to match the design
- Changed hover to white on the cards
- Added top nav for the social media links
- Fixed the width and height of the workshop images

File: about.html [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+        been established to foster and encourage Kenyan artists. The Trust has
+        set up an artists residence by the shores of Lake Naivasha, where a
+        space and accommodation for artists is available. This program is
+        bearing great fruit, and the centre located directly beside the Museum,
+        is open to the public. There are displays of modern art here in all
+        media. Promising artists include Michael Wambua Soi and Rochard Kimathi
+        Wamae.
+      </p>
+      <p class="contact">Please contact us for any further questions at <a href="https://github.com/ginabeki">artsy@studio.com</a> </p>
+    </section>
```
Comment (medaminedev66) [inline about.html:60]: [OPTIONAL] Your layouts on the about page cross each other. I suggest addressing that so your UI design is better.

image

File: README.md [hunk 11]
```diff
@@ -1 +1,86 @@
...
+- Have a local version control like Git. Which is a open source distributed version control system designed for source code management.
+- A text editor (e.g Visual Studio Code, Vim, Atom & Sublime.)
+- A web browser (e.g Chrome, Safari, Mozilla Firefox.)
+- Live Server which is a web extension that helps you to live reload feature for dynamic content.
+- Install wave extension to check project accessibilty.
+
+### Live Demo
+- Here's the [live link](https://www.loom.com/share/87858677e9c740abbdf4e7646deae94a) representation on loom
+
+- To view a live demo of the project you can click on this [link](https://ginabeki.github.io/Conference-Website/).
```
Comment (medaminedev66) [inline README.md:35]: [OPTIONAL] The project is deployed properly. Great job 👏. But in the live demo, I'm still not able to navigate to the home page from the about page. I recommend fixing this issue to improve the user experience.

**Review by medaminedev66 — APPROVED** [review]
Hi @ginabeki 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 💯 
Congratulations! 🎉 

### Highlights
- The video presentation contains a clear and objective description of the project.
- Your design looks great and matches the provided design.

### Optional suggestions
- Check the comments below the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 11 above)
Comment (ginabeki) [reply README.md:35]: @medaminedev66 Thank you for the review, I'll go through the optional suggestions and fix the remaining few areas

**Pull request merged**